### DISCUSSION TITLE
URDF file

### DISCUSSION THREAD

**Discussion by Saketh7256** [Yaskawa-Global/motoros2_client_interface_dependencies#11] [Q&A] [answered]
I'm setting up a MotoROS2 system and need the URDF files for the following robot models to configure everything correctly:

YRC1000micro
HC10DT

Or point me to where I might find them?

Thanks!

**Comment by gavanderhoorn** [comment, marked as answer]
You've probably already found them, but all available support packages can be found at [gavanderhoorn/motoman@ros2](https://github.com/gavanderhoorn/motoman/tree/ros2) for now.

> YRC1000micro

this is a controller, not a robot, and we don't offer a URDF for it.

For future questions related to MotoROS2: could you please post those on the MotoROS2 [Discussion forum](https://github.com/Yaskawa-Global/motoros2/discussions)? That way we keep things centralised.

thanks.
